test/base/TextTest.ooc

```diff
@@ -226,6 +226,17 @@ TextTest: class extends Fixture {
 			text = t"%s %s %i %i" format("test", "number", 0, 1)
 			expect(text == t"test number 0 1")
 		})
+		this add("implicit toText", func {
+			one := t"123" + 456 + 7.89f
```

Comment:
To be fair, one could say the same thing about `value := 2.31f + 7`: Is it a `Float` or `Int`? For those situations we have the rule that `Float` plus anything is always a `Float`, if possible. What we say now is a `Text` plus anything is a `Text`, if possible.